src/darwin/Framework/CHIP/MTROTAProviderDelegate.h

```diff
@@ -16,26 +16,13 @@
  */
 
 #import <Foundation/Foundation.h>
-#import <Matter/Matter.h>
+#import <Matter/MTRCluster.h>
+#import <Matter/MTRCommandPayloadsObjc.h>
+#import <Matter/MTRDefines.h>
```

Comment:
Why this change?

Reply:
The umbrella header imports all public headers, so public headers shouldn't in turn import the umbrella header, but the actual specific other public headers that they need.